backend/src/formatters/timeline_formatter.py

```
@@ -18,7 +18,10 @@ def calculate_checkin_time(flight: Flight) -> str:
             checkin_time = departure_time - timedelta(hours=2)
             return checkin_time.strftime('%H:%M')
         except Exception as e:
-            logger.error(f"Could not calculate check-in time from {flight.departure.time}: {str(e)}")
+            logger.error(
+                f"Could not calculate check-in time from "
+                f"{flight.departure.time}: {str(e)}"
+            )
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)